src/utilities/logging/index.ts

```diff
@@ -30,7 +32,7 @@ export const tidyLogFiles = async () => {
   } catch (error) {
     const storeCode = UIDatabase.getSetting(SETTINGS_KEYS.THIS_STORE_CODE);
     const syncUrl = UIDatabase.getSetting(SETTINGS_KEYS.SYNC_URL);
-    Bugsnag.notify(error, content => {
+    Bugsnag.notify(error as NotifiableError, (content: any) => {
```

Comment:
should have tried harder to type the `content` parameter. Thought it didn't matter much 🤷